lib/game.rb

```diff
@@ -48,6 +52,7 @@ def cpu_setup
     end
     
   def player_setup
+    clear
```

Comment:
I love that you found this native command to clear the terminal.  That is gonna make our display board more clean. Great!